server/src/main/java/org/opensearch/index/translog/transfer/TranslogTransferMetadata.java

```diff
@@ -83,6 +85,7 @@ public String getFileName() {
         return String.join(
             METADATA_SEPARATOR,
             Arrays.asList(
+                METADATA_PREFIX,
```

Comment:
We are not splitting the filename on `METADATA_SEPARATOR` and accessing individual tokens, right?

Reply:
Yes, that is correct .